app/serializers/api/v2/serializable_course.rb

```diff
@@ -67,7 +68,8 @@ def enrichment_attribute(name, enrichment_name = name)
 
       meta do
         {
-          edit_options: @object.edit_options.all
+          edit_options: @object.edit_options.all,
+          start_dates: start_dates(@object.recruitment_cycle.year)
```

Comment:
This should live in the edit_options object.

Reply:
Done.  Thanks for the feedback.